oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/OtelMetricReceiverProvider.java

```diff
@@ -72,12 +82,10 @@ public void start() throws ServiceNotProvidedException, ModuleStartException {
             .provider()
             .getService(GRPCHandlerRegister.class);
         final MeterSystem meterSystem = getManager().find(CoreModule.NAME).provider().getService(MeterSystem.class);
-        final List<Handler> handlers =
-            Handler.all().stream()
-                .filter(h -> config.getEnabledHandlers().contains(h.type()))
-                .collect(toList());
-        for (Handler h : handlers) {
-            h.active(config, meterSystem, grpcHandlerRegister);
+        for (Handler handler : this.handlers) {
+            if (config.getEnabledHandlers().contains(handler.type())) {
+                handler.active(config, meterSystem, grpcHandlerRegister);
+            }
```

Comment:
Why do you need this change? This is a breaking change that cause user-defined handlers not to work anymore

Reply:
Sorry, I want to export `OpenTelemetryMetricHandler` as a service `OpenTelemetryMetricHandlerExporter`.
How about I put `Handler.all()` to prepare stage, and ` handler.active` to start

Reply:
I think you should move part of `OpenTelemetryMetricHandler` logic out of it for reuse, rather than export a handler.
Hanlder could stay for RPC relative logic only.

Reply:
I moved the logic to  `OpenTelemetryMetricRequestProcessor`